python/ray/rllib/rollout.py

```diff
@@ -124,37 +140,46 @@ def rollout(agent, env_name, num_steps, out=None, no_render=True):
     while steps < (num_steps or steps + 1):
         if out is not None:
             rollout = []
-        state = env.reset()
+        obs = env.reset()
```

Comment:
should the mapping cache be reset as well?

Reply:
Yeah I think that's appropriate.